cryptol-remote-api/python/cryptol/bitvector.py

```diff
@@ -178,6 +178,10 @@ def __index__(self) -> int:
         """Equivalent to ``self.value()``."""
         return self.__value
 
+    def __int__(self) -> int:
```

Comment:
Is this needed to make this work with Python 3.7? Or is this a new piece of functionality? I'm not necessary opposed to adding it, but if this is a new feature, perhaps it should belong in its own commit.

Reply:
Ah, that makes sense. Thanks for the clarification!